resources/public/api/conf/1.0/scenarios/deletePensionCharges.md

```diff
@@ -0,0 +1,35 @@
+  <p>Scenario simulations using Gov-Test-Scenario headers is only available in the sandbox environment.</p>
+<table>
+    <thead>
+        <tr>
+            <th>Header Value (Gov-Test-Scenario)</th>
+            <th>Scenario</th>   
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><p>N/A - DEFAULT</p></td>
+            <td><p>Simulates a success response.</p></td>
+        </tr>
+        <tr>
+            <td><p>FORMAT_NINO</p></td>
+            <td><p>Simulates a scenario where the format of the supplied NINO field is not valid.</p></td>
+        </tr>  
+        <tr>
+            <td><p>FORMAT_TAX_YEAR</p></td>
+            <td><p>Simulates a scenario where the format of the supplied tax year field is not valid.</p></td>
+        </tr> 
+        <tr>
+             <td><p>RULE_TAX_YEAR_RANGE_INVALID</p></td>
```

Comment:
RULE_TAX_YEAR_RANGE_INVALID and RULE_TAX_YEAR_NOT_SUPPORTED are not back end errors so won't be in the stub and shouldn't be Test Scenarios as they can be generated at the front end by changing the data passed into the call.